AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main_Optimized.py

```diff
@@ -53,7 +53,7 @@ def wrap_run(cls, instance, request, workspace, editor, editor_test_results, lau
         for f in original_file_list:
             fm._restore_file(f, file_list[f])
 
-@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
+# @pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
```

Comment:
Awesome that we can turn off the xfail, let's remove the line entirely then

Reply:
Sorry we can't remove that yet. I am still doing some debugging right now.

Reply:
Sorry I just accidentally pushed the commented code.
I removed the xfail mark just for debugging.